payment-service/src/main/java/com/example/paymentservice/service/StripeService.java

```diff
@@ -0,0 +1,104 @@
+package com.example.paymentservice.service;
+
+import com.example.paymentservice.exception.BalanceException;
+import com.example.paymentservice.exception.CreateCustomerException;
+import com.example.paymentservice.exception.GenerateTokenException;
+import com.example.paymentservice.exception.PaymentException;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.*;
+import com.stripe.param.CustomerCreateParams;
+import com.stripe.param.CustomerUpdateParams;
+import com.stripe.param.PaymentIntentConfirmParams;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class StripeService {
+
+    @Value("${stripe.secret}")
+    private String SECRET_KEY;
+
+    @Value("${stripe.public}")
+    private String PUBLIC_KEY;
+
+    public Customer createStripeCustomer(CustomerCreateParams customerCreateParams) {
+        try {
+            Stripe.apiKey = SECRET_KEY;
+            return Customer.create(customerCreateParams);
+        } catch (StripeException ex) {
+            throw new CreateCustomerException(ex.getMessage());
+        } finally {
+            Stripe.apiKey = SECRET_KEY;
+        }
```

Comment:
Здесь блок finally co Stripe.apiKey = SECRET_KEY не нужен, лучше переделать метод так (это замечание касается всех остальных методов в StripeService):
```suggestion
       public Customer createStripeCustomer(CustomerCreateParams customerCreateParams) {
        try {
            RequestOptions requestOptions = RequestOptions.builder()
                    .setApiKey(SECRET_KEY)
                    .build();

            return Customer.create(customerCreateParams, requestOptions);
        } catch (StripeException ex) {
            throw new CreateCustomerException(ex.getMessage());
        }
    }
```